content/en/docs/reference/glossary/priority-class.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Priority Class
+id: priority-class
+date: 2023-07-10
+full_link: /docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass
+short_description: >
+ The PriorityClass is a resource that allows you to assign priorities to different pods within a cluster It helps the scheduler prioritize the scheduling of pods based on their importance or criticality.
```

Comment:
This is a bit long. Can you write just one sentence here?

Reply:
> a cluster It helps

Well, now it's one sentence, but only because punctuation was removed. ;)

I think it would be good to just stop at "within a cluster." and drop the second sentence. That's enough for a short description.